src/main/java/core/basesyntax/service/impl/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> readFromFile(String filePath) {
+        List<String> strings;
+        try {
+            strings = Files.readAllLines(Path.of(filePath)).stream()
+                    .skip(1)
```

Comment:
Why do we skip one line here? If it a specific for SCV file we should name our class not just a FileReaderService. And also better make it constant